lib/wsdl.js

```diff
@@ -1957,5 +1975,13 @@ function open_wsdl(uri, options, callback) {
   return wsdl;
 }
 
+function clear_cache() {
+  WSDL_CACHE = {};
```

Comment:
instead of assigning a new object, I feel you should delete all the properties using `Object.keys(WSDL_CACHE).forEach(key) {delete WSDL_CACHE[key];});` or something similar to avoid stale references from lingering around.